src/components/plans/ChargeAccordion.tsx

```diff
@@ -854,7 +854,8 @@ export const ChargeAccordion = memo(
                   </Button>
                 ) : (
                   <SpendingMinimumWrapper>
-                    <SpendingMinimumInput
+                    <AmountInput
+                      className="flex-1"
```

Comment:
Maybe not something we want to tackle not but felt worth mentioning: all inputs (and generally all elements in our design system) should take the maximum width by default. And only when needed we can define a custom max-width.
Feel free to ignore as it may lead to many check and fixes to migrate now :)